packages/template-retail-react-app/app/static/dwac-21.7.js

```diff
@@ -0,0 +1,470 @@
+// Enhance the dw.ac namespace
+(function(dw) {
```

Comment:
immediately invoked function expressions (IIFE) like this are generally not friendly with React since the mounting / loading mechanism creates a race condition between two separate frameworks (e.g. the global javascript / window namespace that is used here and, in conflict with it, the React lifecycle / DOM mounting mechanism).

It's preferable not to approach this functionality in a global "non-React" way as we are doing in this file

Reply:
Again, I suppose this is part of a third party library, the problem is this wasn't clear because the PR description (and also code comments) lack any reference to the 3rd party library source. Please add reference for future visibility